packages/ui/components/ui/tx/viewbox.tsx

```diff
@@ -11,44 +11,43 @@ export interface ViewBoxProps {
   isOpaque?: boolean;
 }
 
-const Label = ({ label }: { label: string }) => <span className='text-lg'>{label}</span>;
+const Label = ({ label }: { label: string }) => (
+  <span className='text-lg font-bold text-gray-300'>{label}</span>
+);
 
 export const ViewBox = ({ label, visibleContent, isOpaque }: ViewBoxProps) => {
   // if isOpaque is undefined, set it to !visibleContent
-  isOpaque = isOpaque ?? !visibleContent;
+  isOpaque = isOpaque ?? !visibleContent?.props;
+
   return (
     <Box overflow='hidden'>
-      <div
-        className={cn(
-          'flex flex-col gap-1 break-all overflow-hidden',
-          isOpaque ? 'cursor-not-allowed' : '',
-        )}
-      >
-        <div className='flex items-center gap-2 self-start'>
-          <span className={cn('text-base font-bold', isOpaque ? 'text-gray-600' : '')}>
-            {!isOpaque && <Label label={label} />}
-            {isOpaque && (
-              <div className='flex gap-2'>
+      <div className={cn('flex flex-col gap-2', isOpaque ? 'cursor-not-allowed' : '')}>
+        <div className='flex items-center gap-2'>
+          <span className={cn('text-base', isOpaque ? 'text-gray-600' : '')}>
+            {isOpaque ? (
+              <div className='flex items-center gap-2'>
                 <IncognitoIcon fill='#4b5563' />
                 <Label label={label} />
               </div>
+            ) : (
+              <Label label={label} />
             )}
           </span>
         </div>
-        {visibleContent}
+        {visibleContent && <div className='border-t border-gray-700 pt-2'>{visibleContent}</div>}
```

Comment:
suggestion: personally, I think you can get rid of the horizontal line border

Reply:
I actually prefer the horizontal line border because it provides clearer visual separation and enhances the structure.

<img width="512" alt="Screenshot 2024-11-25 at 1 33 02 PM" src="https://github.com/user-attachments/assets/b1371000-bf58-4050-84c6-33d38901f0bb">

<img width="512" alt="Screenshot 2024-11-25 at 1 32 24 PM" src="https://github.com/user-attachments/assets/210241bb-eeea-42b9-92e8-9655570ee22a">